docs/en/observability/logs-index-template.asciidoc

```diff
@@ -0,0 +1,54 @@
+[[logs-index-template]]
+= Logs index template reference
+
+[[custom-logs-index-template]]
+== View index templates
+
+Index templates are used to configure the backing indices of data streams as they are created.
+These index templates are composed of multiple component templates--reusable building blocks
```

Comment:
Can we link to the Elasticsearch docs for component templates here for users that are interested in more details?